utils/update_checkout/update_checkout/update_checkout.py

```diff
@@ -18,6 +18,11 @@
 import sys
 import traceback
 
+
+SCRIPT_FILE = os.path.abspath(__file__)
+SCRIPT_DIR = os.path.dirname(SCRIPT_FILE)
+sys.path.append(os.path.dirname(os.path.dirname(SCRIPT_DIR)))
+
```

Comment:
Is this for fixing the test `update_checkout.test-sh`? I changed the test instead to provide a working directory with the `-t` parameter.

```
# RUN: %{python} -m unittest discover -s %swift_src_root/utils/update_checkout -t %swift_src_root/utils/
```

Reply:
This was to deal with importing swift_build_support, which lives one directory up from update-checkout and wasn't found by the default paths in 3.

Reply:
Yes, I understand why you add it. In my tests with Python 3, when running the script by itself, I didn’t need any changes here. But when running from the tests, I found out that the modules weren’t in the search path. I used the solution I pointed above, since it doesn’t modify the code for all cases.